package.json

```diff
@@ -37,6 +37,7 @@
     "vue-router": "^3.0.0",
     "vuex": "^3.0.1",
     "vuex-persistedstate": "^2.4.2",
+    "vuex-router-sync": "^5.0.0",
```

Comment:
what is this needed for?

Reply:
Hmm, sorry, apparently I reintroduced this dependency while merging

Reply:
Fixed with rebase and force push!